frontend/public/components/monitoring/alert-manager-receiver-forms.tsx

```diff
@@ -446,17 +451,37 @@ export const CreateReceiver = () => (
   </Firehose>
 );
 
+export const EditReceiver = ({ match: { params } }) => {
+  return (
+    <Firehose
+      resources={[
```

Comment:
Probably make `resources` a top level `const` since it's identical for `CreateReceiver` and `EditReceiver`?

Reply:
thanks, fixed